sdk/servicebus/azure-servicebus/README.md

```diff
@@ -13,8 +13,8 @@ Use the Service Bus client library for Python to communicate between application
 
 [Source code](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/servicebus/azure-servicebus/) | [Package (PyPi)][pypi] | [API reference documentation][api_docs] | [Product documentation][product_docs] | [Samples](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/servicebus/azure-servicebus/samples) | [Changelog](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/servicebus/azure-servicebus/CHANGELOG.md)
 
-> **NOTE**: This document has instructions, links and code snippets for the **preview** of the next version of the `azure-servicebus` package
-> which has different APIs than the current version (0.50). Please view the resources below for references on the existing library.
+> **NOTE**: This document has instructions, links and code snippets for the newly-released GA version of the `azure-servicebus` package.
+> The prior preview version (0.50) has different APIs and is now deprecated. Please view the resources below for if historical references are needed, but migration is recommended.
```

Comment:
I don't think we are supposed to use the word 'deprecated'. You can take out the phrase "and is now deprecated". Maybe restate final sentence as "Please view the resources below for reference, but migration is recommended."

Reply:
Synced with Elena OOB on your guidance, got her excellent verbiage here.  Shout if any issues.